libs/tools/export/vault-export/vault-export-core/src/services/individual-vault-export.service.ts

```diff
@@ -169,9 +172,13 @@ export class IndividualVaultExportService
       }),
     );
 
+    const restrictions = await firstValueFrom(this.restrictedItemTypesService.restricted$);
+
     promises.push(
       this.cipherService.getAllDecrypted(activeUserId).then((ciphers) => {
-        decCiphers = ciphers.filter((f) => f.deletedDate == null);
+        decCiphers = ciphers
+          .filter((f) => f.deletedDate == null)
+          .filter((c) => !ServiceUtils.isCipherRestricted(c, restrictions));
```

Comment:
⛏️ This can be combined into a single filter

```suggestion
          .filter((f) => f.deletedDate == null && !ServiceUtils.isCipherRestricted(c, restrictions));
```